resources/metadata/kubernetesDeploy.yaml

```diff
@@ -279,6 +279,14 @@ spec:
           - STAGES
           - STEPS
         default: 300
+      - name: helmTestWaitSeconds
+        type: int
+        description: Time to wait for any individual Kubernetes operation (like Jobs for hooks) . this param gets translated to `--timeout param` for helm cli during helm test.
+          Default value is 5m0s (5 min and 0 seconds) if not set via parameter. See https://helm.sh/docs/helm/helm_test/#options for further details
```

Comment:
Since a user can provide time only in seconds, probably it makes sense to reflect it in the documentation explicitly.
`Time to wait for any individual…` --> `Time in seconds to wait for any individual…`.?